README.md

```diff
@@ -1,19 +1,60 @@
-<p align="center">
-  <a href="https://sentry.io" target="_blank" align="center">
-    <img src="https://sentry-brand.storage.googleapis.com/sentry-logo-black.png" width="280">
-  </a>
-  <br />
+<p  align="center">
+	<a  href="https://sentry.io"  target="_blank"  align="center">
+		<img  src="https://sentry-brand.storage.googleapis.com/sentry-logo-black.png"  width="280">
+	</a>
+	<br/>
 </p>
 
-Sentry SDK for Capacitor
-===========
+## Official Sentry SDK for Capacitor
 
 _Work in progress_ Sentry SDK for [Capacitor](https://capacitorjs.com/).
 
+**Release Stage**
+_Android alpha_
+
 #### Resources
 
-* [![Documentation](https://img.shields.io/badge/documentation-sentry.io-green.svg)](https://docs.sentry.io/platforms/capacitor/)
-* [![Forum](https://img.shields.io/badge/forum-sentry-green.svg)](https://forum.sentry.io/c/sdks)
-* [![Discord](https://img.shields.io/discord/621778831602221064)](https://discord.gg/Ww9hbqr)
-* [![Stack Overflow](https://img.shields.io/badge/stack%20overflow-sentry-green.svg)](https://stackoverflow.com/questions/tagged/sentry)
-* [![Twitter Follow](https://img.shields.io/twitter/follow/getsentry?label=getsentry&style=social)](https://twitter.com/intent/follow?screen_name=getsentry)
+[![Documentation](https://img.shields.io/badge/documentation-sentry.io-green.svg)](https://docs.sentry.io/platforms/capacitor/)
```

Comment:
We usually have `Resources` at the bottom of the readme, i.e:

https://github.com/getsentry/sentry-react-native